release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2855,6 +2855,53 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.17.4
+[id="ocp-4-17-4_{context}"]
+=== RHSA-2024:XXXX - {product-title} {product-version}.4 bug fix, and security update advisory
+
+#Included for placement#
+
+Issued: 13 November 2024
+
+#Included for placement#
+
+{product-title} release {product-version}.4, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:XXXX[RHSA-2024:XXXX] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:XXXX[RHSA-2024:XXXX] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.17.4 --pullspecs
+----
+
+#Included for placement#
+
+[id="ocp-4-17-4-enhancements_{context}"]
+==== Enhancements
+
+[id="ocp-4-17-4-enhancements-nodes-pods-short-term-auth_{context}"]
+===== Authenticating customer workloads with {gcp-wid-short}
+
+With this release, applications in customer workloads on {product-title} clusters that use {gcp-wid-first} can authenticate by using {gcp-wid-short}.
+
+To use this authentication method with your applications, you must complete configuration steps on the cloud provider console and your {product-title} cluster.
```

Comment:
```suggestion
To use this authentication method with your applications, you must configure the cloud provider console and your {product-title} cluster.
```
Would "configure" mean the same thing as "complete configuration steps" here?

Reply:
thanks! they are configuring IAM things by using the console, but the full docs get into that detail